File: tests/test_files/js/invalid.js
// Missing proper META and SYNOPSIS blocks

//// meta: Invalid lowercase
/// META: Wrong prefix level
//// META:Missing space after colon
///// META: Too many slashes

/// SECTION: incomplete-section
// Missing proper closure
const testFunction = () => {
    return true;
}
// Missing END tag

// Invalid continuation
//// TODO: Multi-line
/// continuation with wrong prefix

//// METRICS: invalid format 90
/// REVIEW:Wrong prefix level

File: tests/test_files/c/invalid.c
// Missing proper header block
//// meta: lowercase tag
///META: Invalid spacing
//// META:Missing space after colon
///// META: Too many slashes

/// SECTION: test-section
// Missing required header blocks
void test_function() {
    return;
}
/// END: SECTION: mismatched-name  // Section name mismatch

// Invalid tag formats
//// TODO Missing colon
//// METRICS Invalid format 100
/// REVIEW:Wrong prefix level
